pp/requests/requests.go

```diff
@@ -115,6 +115,58 @@ func ReqGetWalletOzData(walletAddr string) *protos.ReqGetWalletOz {
 	}
 }
 
+// RequestUploadFile request upload file without locating the file at the "path"
+func RequestUploadFile(fileName, fileHash string, fileSize uint64, reqID, ownerWalletAddress string, isEncrypted bool) *protos.ReqUploadFile {
+	utils.Log("fileName~~~~~~~~~~~~~~~~~~~~~~~~", fileName)
```

Comment:
don't use "~" for the log, previous code of log like this need to be cleaned later